src/types.ts

```diff
@@ -0,0 +1,21 @@
+import type {
+  OpenFramesTrustedData,
+  OpenFramesUntrustedData,
+} from '@open-frames/types';
+
+export type DscvrUntrustedData = OpenFramesUntrustedData & {
+  dscvrId: string;
+  contentId?: string;
+  state?: string;
+};
+export const dscvrClientProtocolPrefix = 'dscvr@' as const;
```

Comment:
shouldn't this be `src/constants.ts` as `src/types.ts` should only export types & should be used like:

`import type { <type> } from './types';`

Reply:
Done